src/components/textarea/textarea.component.tsx

```diff
@@ -264,7 +264,7 @@ export const Textarea = React.forwardRef(
 
         const scrollPosition = scrollElement?.scrollTop;
 
-        textarea.style.height = "0px";
+        if (!rows && expandable) textarea.style.height = "0px";
```

Comment:
**suggestion(non-blocking)**: it appears this line was added a while ago to get round some default behaviour in IE. Given we don't support IE anymore, what do you think about removing this?


```suggestion
```

Reply:
Yeah, I saw it was a legacy thing when I investigated why it was in the code in the first place. I can delete it and see if anything breaks. If nothing breaks, I'll happily remove it 😄